app/views/suggestions/_suggestion.html.erb

```diff
@@ -0,0 +1,34 @@
+<% if @suggestions.class == Array %>
+  <% @suggestions.each do |hash|%>
+
+  <div class = "suggestion">
+    <% if hash.class == TunesTakeoutWrapper %>
+      <% hash = hash.suggestion %>
+    <%end%>
+    Eat at
+    <% food = hash["food_suggestion"]%>
+    <%= link_to food.name, "https://yelp.com/biz/#{hash["food_id"]}" %><%= image_tag "#{food.rating}" %>
+    <section class = "float-right"><%= image_tag "#{food.image}" %><%= food.address.address[0]%></section>
+    <br />
+    While listening to the <%= hash["music_type"]%>
+    <% sound = hash["music_suggestion"]%>
+    <%= link_to sound.name, sound.url %>
+    <section class = "float-right covers"><%= (sound.cover.class == Array) ? (image_tag "#{sound.cover[0]["url"]}") : (image_tag "#{sound.cover["url"]}") %> <%= food.address.address[0]%></section>
+    <div class = "fave-button">
+      <% if current_user %>
+        <% user_faves = TunesTakeoutWrapper.user_favorites(current_user.uid)%>
+        <% unless user_faves ==[]%>
+          <% user_faves = user_faves.suggestions%> #returns array of pairing id's %>
+        <%end%>
+        <% if user_faves.include? hash["id"] %>
+          <%= link_to "Unfavorite this pairing", unfave_path(hash["id"], @path), method: :delete, class: "btn btn-danger" %>
+        <% else%>
+        <%= link_to "Favorite this pairing", add_fave_path(hash["id"], @path), method: :post, class: "btn btn-primary" %>
+        <%end%>
+      <%end%>
+    </div>
+    </div>
+  </div>
+  <br />
+  <%end%>
```

Comment:
A decent amount of the logic in this view could happen within a controller or helper method. Your HTML should be as clean as possible